ledger/src/blockstore.rs

```diff
@@ -3441,6 +3457,30 @@ fn handle_chaining(
     Ok(())
 }
 
+/// A helper function of handle_chaining which handles the chaining based
+/// on the `SlotMetaWorkingSetEntry` of the specified `slot`.  Specifically,
+/// it handles the following two things:
+///
+/// First, based on the `SlotMetaWorkingSetEntry` of the input `slot`, it it
+/// indicates the slot did not have a parent slot before but now has a parent
+/// slot, then it will update the Orphan property of both the slot and its
+/// parent slot based on their current orphan status.
+///
+/// Second, if the `SlotMetaWorkingSetEntry` of the input `slot` indicates this
```

Comment:
nit: Similar to above, think you can drop "the input" and change "of" to "for"
==> "Second, if the `SlotMetaWorkingSetEntry` for `slot` indicates it is newly ..."